sphinx_wagtail_theme/domainindex.html

```diff
@@ -40,7 +40,7 @@ <h1>{{ indextitle }}</h1>
      <tr{% if grouptype == 2 %} class="cg-{{ groupid.current() }}"{% endif %}>
        <td>{% if grouptype == 1 -%}
          <img src="{{ pathto('_static/minus.png', 1) }}" class="toggler"
-              id="toggle-{{ groupid.next() }}" style="display: none" alt="-" />
+              id="toggle-{{ groupid.next() }}" style="display: none" alt="-" decoding="async" loading="lazy" />
```

Comment:
I think this image should not be async lazy as we need this to show instantly on load as it's a menu button.